### DISCUSSION TITLE
Link tag is not adding href attribute to anchor tag. Published my own link component using next's link inside the storybook and when importing it inside another project href attribute is not getting attached.

### DISCUSSION THREAD

**Discussion by ajayjaggi97** [vercel/next.js#30446] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.0.0

### What browser are you using?

Safari

### What operating system are you using?

macos

### How are you deploying your application?

next start

### Describe the Bug

No href attribute present on anchor tag.
![Uploading Screenshot 2021-10-27 at 2.52.03 PM.png…]()

### Expected Behavior

Href attribute to be present

### To Reproduce

Implement Link component in storybook repo. 
Publish it.
Use in another project

// implementation of link component

`import React, { FC } from 'react';
import NextLink, { LinkProps as NextLinkProps } from 'next/link';

interface LinkProps extends NextLinkProps {
  className?: string;
  target?: string;
}

const Link: FC<LinkProps> = (props) => {
  const { children, className = '', target = '_self', href, ...rest } = props;

  return (
    <NextLink {...rest} href={href ?? ''}>
      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
      <a className={className} target={target} >
        {children}
      </a>
    </NextLink>
  );
};

export { Link };
export type { LinkProps };
`

**Comment by ijjk** [comment]
Hi, you need to use `passHref` when using a custom component inside of `next/link` instead of the recommended `a` tag. Related documentation can be found here https://nextjs.org/docs/api-reference/next/link
